src/components/List/List.tsx

```diff
@@ -1,37 +1,44 @@
-import { listMockData } from './listMockData';
 import { NavLink } from 'react-router-dom';
 import './List.scss';
+import { GardenKeys } from '../Main/Main';
+import { CellKeys } from '../Grid/Grid';
 
-type Plant = {
-  'plant_name': string,
-  'plant_id': number,
-  'image': string
-};
+// type Plant = {
+//   'plant_name': string,
+//   'plant_id': number,
+//   'image': string
+// };
 
-const List = () => {
-  const findQuantity = (array: Plant[], id: number): number => {
-    let count = 0;
-    for (const element of array) {
-      if (element['plant_id'] === id) {
-        count++;
+interface ListProps {
+  garden: GardenKeys | undefined;
+}
+
+const List = ({ garden }: ListProps) => {
+  const findQuantity = (array: GardenKeys | undefined, id: number | undefined): number | undefined => {
+    if (array) {
+      let count = 0;
+      for (const element of array) {
+        if (element['plant_id'] === id) {
+          count++;
+        }
       }
+      return count;
```

Comment:
This likely will be resolved with the new BE endpoints, but please leave it until you're able to check when you connect to it!